syntax = "proto3";
package collector;




import "model/inspection/collector.proto";

/*
ListCollector请求
*/
message ListCollectorRequest {
    /*
    套件id
    */
    string id = 1 ;
    /*
    页码
    */
    int32 page = 2 ;
    /*
    每页大小
    */
    int32 pageSize = 3 ;
}

/*
ListCollector返回
*/
message ListCollectorResponse {
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated inspection.InspectionCollector list = 4 ;
}

/*
ListCollectorApi返回
*/
message ListCollectorResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    collector.ListCollectorResponse data = 4 ;
}

